index.js

```diff
@@ -5,7 +5,26 @@ const OSC = '\u001B]';
 const BEL = '\u0007';
 const SEP = ';';
 
-const isTerminalApp = process.env.TERM_PROGRAM === 'Apple_Terminal';
+/* global window */
+const isBrowser = typeof window !== 'undefined' && typeof window.document !== 'undefined';
+
+let isTerminalApp_;
+let platform_ = 'browser';
+let cwdFunc_ = () => {
+	throw new Error('NodeJS.Process.cwd() doesn\'t work in Browser.');
+};
+
+if (isBrowser) {
+	isTerminalApp_ = false;
+} else {
+	isTerminalApp_ = process.env.TERM_PROGRAM === 'Apple_Terminal';
+	platform_ = process.platform;
+	cwdFunc_ = process.cwd;
+}
+
+const isTerminalApp = isTerminalApp_;
+const platform = platform_;
+const cwdFunc = cwdFunc_;
```

Comment:
Would be simpler to just use a ternary for each of these. Then you don't need all this reassignment.